templates/typescript/samples/sample-assistant/src/inversify.config.ts

```diff
@@ -0,0 +1,246 @@
+        );
+        const allowedCallersClaimsValidator: AllowedCallersClaimsValidator = new AllowedCallersClaimsValidator(container.get<SkillsConfiguration>(TYPES.SkillsConfiguration));
+
+        // Register AuthConfiguration to enable custom claim validation.
+        container.bind<AuthenticationConfiguration>(TYPES.AuthenticationConfiguration).toConstantValue(
+            new AuthenticationConfiguration(undefined,(claims: Claim[]) => allowedCallersClaimsValidator.validateClaims(claims))
+        );
+
+        const skillDialogs: SkillDialog[] = skills.map((skill: IEnhancedBotFrameworkSkill): SkillDialog => {
+            const skillDialogOptions: SkillDialogOptions = {
+                botId: appsettings.microsoftAppId,
+                conversationIdFactory: container.get<SkillConversationIdFactoryBase>(TYPES.SkillConversationIdFactory),
+                skillClient: container.get<SkillHttpClient>(TYPES.SkillHttpClient),
+                skillHostEndpoint: hostEndpoint,
+                skill: skill,
+                conversationState: container.get<ConversationState>(TYPES.ConversationState)
+            };
+            return new SkillDialog(skillDialogOptions, skill.id);
+        });
+
+        decorate(injectable(), SkillDialog);
+        container.bind<SkillDialog[]>(TYPES.SkillDialogs).toConstantValue(skillDialogs);
+    }
+}
+
+if (!container.isBound(TYPES.AuthenticationConfiguration)) {
+    container.bind<AuthenticationConfiguration>(TYPES.AuthenticationConfiguration).toConstantValue(
+        new AuthenticationConfiguration(undefined)
+    );
+}
+
+// Configure bot
+decorate(injectable(), ActivityHandlerBase);
+decorate(injectable(), ActivityHandler);
+decorate(injectable(), TeamsActivityHandler);
+decorate(injectable(), DefaultActivityHandler);
+container.bind<DefaultActivityHandler<MainDialog>>(TYPES.DefaultActivityHandler).to(DefaultActivityHandler).inTransientScope();
+
+// Configure SkillHandler
+decorate(injectable(), ChannelServiceHandler);
+decorate(injectable(), SkillHandler);
+
+container.bind<SkillHandler>(TYPES.SkillHandler).toConstantValue(new SkillHandler(
+    container.get<DefaultAdapter>(TYPES.DefaultAdapter),
+    container.get<DefaultActivityHandler<MainDialog>>(TYPES.DefaultActivityHandler),
+    container.get<SkillConversationIdFactoryBase>(TYPES.SkillConversationIdFactory),
+    container.get<SimpleCredentialProvider>(TYPES.SimpleCredentialProvider),
+    container.get<AuthenticationConfiguration>(TYPES.AuthenticationConfiguration)
+));
+
+export default container;
+
+function getTelemetryClient(settings: Partial<IBotSettings>): BotTelemetryClient {
+    if (settings !== undefined && settings.appInsights !== undefined && settings.appInsights.instrumentationKey !== undefined) {
+        const instrumentationKey: string = settings.appInsights.instrumentationKey;
+
+        return new ApplicationInsightsTelemetryClient(instrumentationKey);
+    }
+
+    return new NullTelemetryClient();
+}
```

Comment:
```suggestion
function getTelemetryClient(settings: Partial<IBotSettings>): BotTelemetryClient {
    if (settings !== undefined && settings.appInsights !== undefined && settings.appInsights.instrumentationKey !== undefined) {
        const instrumentationKey: string = settings.appInsights.instrumentationKey;

        return new ApplicationInsightsTelemetryClient(instrumentationKey);
    }

    return new NullTelemetryClient();
}

export default container;
```

nit: because the last thing to do in this file is to export the `container`